package.json

```diff
@@ -10,7 +10,8 @@
     "premigrations": "npx sequelize-cli db:migrate --env preMigrations --migrations-path util/db/pre-migrations",
     "migrations": "npx sequelize-cli db:migrate  --env database",
     "prestart": "npm run lint && npm run migrations",
-    "start": "node --experimental-modules src/server.js"
+    "start": "node --experimental-modules src/server.js",
+    "nm:run": "newman run tests/trap-registration-api.postman_collection.json"
```

Comment:
Can you add a test command here using the `start-server-and-test` module like you did in Scottish-Natural-Heritage/trap-registration#46.

Reply:
Resolved in ab3b133c3cbba8efde02a7db5037014eb67bff4d.